wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md

```diff
@@ -1,33 +1,36 @@
+Changing Title Text
+=====================
+
 [*Changing the Title Text : The Guide* by: Ekaru](https://osu.ppy.sh/forum/t/14513)
 
 How to get it changed
-=====================
+-------------------------
 
 Sadly, you can't do this yourself. Just ask a QAT to change it. They're the only ones that are able to, so just give them a note of what you need/want, and if it's reasonable enough, you'll get your new title text on your map! Easy, right?!
 
 Why change?
```

Comment:
L11: Suspected the grammar here to be too simple. Suggested: ``Why/When do I need to change?``